### PR TITLE
only run DST helper on certain pumps

### PR DESCRIPTION
**Pull request by AdrianLxM**

### CODE REVIEW

**Comment by TebbeUbben** [comment]
Shouldn't we rather add this field to the pump description?

**Comment by AdrianLxM** [comment]
Pump descriptions don't distinguish if it is a real pump or a virtual pump.
It is rather the pump driver that can or can't handle it.

File: app/src/main/java/info/nightscout/androidaps/plugins/constraints/dstHelper/DstHelperPlugin.java [hunk 1]
```diff
@@ -43,83 +42,53 @@ public DstHelperPlugin() {
...
         }
+
+        if (!value.value()) {
+            log.debug("Already not allowed - don't check further");
+            return value;
+        }
+
+        if (wasDST(cal)) {
+            LoopPlugin loopPlugin = LoopPlugin.getPlugin();
+            if (loopPlugin.isSuspended()) {
```
Comment (schmitzn) [inline app/src/main/java/info/nightscout/androidaps/plugins/constraints/dstHelper/DstHelperPlugin.java]: I think it should be:
```suggestion
            if (!loopPlugin.isSuspended()) {
```

**Comment by MilosKozak** [comment]
👍

**Pull request merged**